### DISCUSSION TITLE
How can I link the paper nodes to their title and abstract?

### DISCUSSION THREAD

**Discussion by jiaruHithub** [snap-stanford/ogb#324] [General]
Hi, thank you for your construction of liberary.
I want link some nodes of academic graph dataset such as ```ogbn-arxiv```, ```ogbn-paper100M``` and ```ogbl-catition``` to their own title, abstract and authors.
How should I do?

**Comment by weihua916** [comment]
Hi! You can get the text of title and abstract of [ogbn-papers100M](https://github.com/snap-stanford/ogb/tree/master/examples/nodeproppred/papers100M) and [ognn-arxiv](https://github.com/snap-stanford/ogb/tree/master/examples/nodeproppred/arxiv). All the mappings to the MAG database are provided in the `mapping/` directory of the downloaded dataset folder.

**Reply by jiaruHithub** [reply]
Thanks! But I want get the text of title and abstract of  ```Link-Prediction Dataset``` such as ```ogbn-arxiv```. Is there any methods? If not, is there ant question building by myself?

**Reply by weihua916** [reply]
ogbn-arxiv is a node classification data. If you want to access to the text for the other MAG-based dataset, such as ogbl-citation, you will need to crawl the MAG yourself. The instruction is in `mapping/README.md`

**Reply by jiaruHithub** [reply]
Thanks, the mean is that I can crawl the text for the other MAG-based dataset by replacing the number in (https://academic.microsoft.com/paper/2894175828) with the specified paper id in the ```nodeidx2paperid.csv.gz```. right?

**Reply by jiaruHithub** [reply]
> ogbn-arxiv is a node classification data. If you want to access to the text for the other MAG-based dataset, such as ogbl-citation, you will need to crawl the MAG yourself. The instruction is in `mapping/README.md`

I find that the Microsoft Academic has been stopped. The title and abstract does not seem to be crawled, is there any other methods?Thanks

**Reply by weihua916** [reply]
That's right. For now, I do not think there is no easy way to crawl the MAG. That is really out of our control.

**Reply by jiaruHithub** [reply]
OK, thanks, is it mean that there is no easy way to crawl the MAG?
